### ISSUE TITLE
Histograms with only a single unique value come out empty

### ISSUE THREAD

**Issue by jkleint** [Bergvca/pyspark_dist_explore#7]
Thanks for such a great library.

I noticed that when my column only has a single unique value, the histogram doesn't show anything.  Seems like there aren't 10 bins/edges coming out as straight matplotlib does.

```
df = spark.createDataFrame(pd.DataFrame({'a': [1,1,1]}))
fig, ax = plt.subplots(1, figsize=(6,4))
pyspark_dist_explore.hist(ax, df.select('a'))
```

```
(array([3.]), array([1, 1]), <a list of 1 Patch objects>)
```

![image](https://user-images.githubusercontent.com/931505/54499370-8ac6db00-48d6-11e9-9463-141a39f837ce.png)

**Comment by Bergvca** [comment]
Interesting! It seems like in this case numpy/matplotlib just adds a range parameter with 0.5 before and 0.5 after the unique value. I'll update it to mimic this behaviour.

**Comment by Bergvca** [comment]
I finally had some time to work on this, fixed this issue in the following branch:

https://github.com/Bergvca/pyspark_dist_explore/tree/single_value_hist

Let me know what you think! I will pull into master and push to pypi soon.

**Comment by Bergvca** [comment]
This is fixed in 0.1.7 which is now on Pypi
